src/urllib3/util/retry.py

```diff
@@ -159,13 +209,13 @@ class Retry(object):
         request.
     """
 
-    DEFAULT_METHOD_WHITELIST = frozenset(
+    DEFAULT_METHOD_ALLOWLIST = frozenset(
         ["HEAD", "GET", "PUT", "DELETE", "OPTIONS", "TRACE"]
     )
 
     RETRY_AFTER_STATUS_CODES = frozenset([413, 429, 503])
 
-    DEFAULT_REDIRECT_HEADERS_BLACKLIST = frozenset(["Authorization"])
+    DEFAULT_REMOVE_HEADERS_ON_REDIRECT = frozenset(["Authorization"])
```

Comment:
If the name isn't set in concrete, I'd suggest `DEFAULT_HEADER_REMOVELIST` which matches `DEFAULT_METHOD_ALLOWLIST` better and doesn't feel unnecessarily wordy

Reply:
None of the new names are set, these were my first attempt :) I picked the name to match the parameter `remove_headers_on_redirect`.

Reply:
Yeah, I think the redirect bit is important. You're right. Probably should be something like `DEFAULT_HEADER_REMOVELIST_ON_REDIRECT`? Ugh, I don't like that much. `DEFAULT_REDIRECT_HEADER_REMOVELIST`? I think we want to convey as much information as possible. Then again, that's just changing one word there from what we have now. I don't mind rewording it altogether

Reply:
Naming continues to be the scourge of software. :skull_and_crossbones:

Reply:
I like the current name. In a vacuum, I can imagine this being called "headers to remove on redirect", and we're prefixing DEFAULT_ to it. I agree it's less symmetrical to something like `DEFAULT_ALLOWED_METHODS` but that's okay.